python/mxnet/rnn/rnn_cell.py

```diff
@@ -788,6 +788,12 @@ def unpack_weights(self, args):
     def pack_weights(self, args):
         return _cells_pack_weights(self._cells, args)
 
+    def reset(self):
+        super(SequentialRNNCell, self).reset()
+        if hasattr(self, '_cells'):
```

Comment:
Alternatively you can put this in the BaseRNNCell so that other container cells (e.g. BidirectionalCell) can be patched up together.
Would you mind doing the same for python/mxnet/gluon/rnn/rnn_cell.py?